ext/HealthBaseOMOPCDMExt.jl

```diff
@@ -0,0 +1,351 @@
+module HealthBaseOMOPCDMExt
+
+using HealthBase
+using DataFrames
+using OMOPCommonDataModel
+using Serialization
+using InlineStrings
+using Dates
+using Statistics
+import FeatureTransforms: 
+    OneHotEncoding, apply_append
+using DuckDB               
+using DBInterface: execute
+
+# NOTE: In the future, replace this with OMOP CDM version info directly from OMOPCommonDataModel.jl dependencies.
+const OMOPCDM_VERSIONS = deserialize(joinpath(@__DIR__, "..", "assets", "version_info"))
+
+# Mapping OMOP CDM datatypes to Julia types
+const datatype_map = Dict(
```

Comment:
Could you make the constant variable all CAPS? It's more the standard convention within Julia BlueStyle https://github.com/JuliaDiff/BlueStyle